101-azure-machine-learning-service-workspace/azuredeploy.json

```diff
@@ -0,0 +1,123 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Azure Machine Learning service workspace."
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
```

Comment:
remove the spaces from allowedValues and make all lower case

Reply:
Done